kernel/sched.c

```diff
@@ -9,6 +9,7 @@
 #include <zephyr/kernel/internal/sched_priq.h>
 #include <wait_q.h>
 #include <kswap.h>
+#include <kthread.h>
```

Comment:
I'm guessing that the inclusion of the kthread.h header file and the blank line below are artefacts of the development process and should likely be cleaned up.